docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1814,3 +1814,33 @@ The connector supports redirection from Iceberg tables to Hive tables with the
 
 The connector supports configuring and using [file system
 caching](/object-storage/file-system-cache).
+
+### In-memory Iceberg metadata caching
+
+The connector supports caching iceberg metadata in coordinator memory. This
+caching is enabled by default. To disable in-memory Iceberg metadata caching,
+you must set the `iceberg.metadata-cache.enabled` configuration property to
+`false`.
+
+Additionally, you can use the following catalog configuration properties to
+adjust in-memory Iceberg metadata caching:
+
+:::{list-table} In-memory Iceberg metadata caching configuration properties
+:widths: 25, 75
+:header-rows: 1
+
+* - Property
+  - Description
+* - `fs.memory-cache.ttl`
+  - The maximum duration to keep files in the cache prior to eviction. Time
+    starts the moment an entry is saved in the cache. The minimum value of `0s`
+    means that caching is effectively turned off. Defaults to `1h`.
+* - `fs.memory-cache.max-size`
+  - The maximum total size of the cache. This value can be set higher for a
+    large number of tables if you have sufficient coordinator memory. Defaults to
```

Comment:
Reword the sentence to just say something like "When raising this value .. keep in mind that the coordinator memory is used.."